sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -244,6 +260,9 @@ Train a machine-learned model on your own form type. The resulting model will be
 Provide a container SAS url to your Azure Storage Blob container where you're storing the training documents. See details on setting this up
 in the [service quickstart documentation][quickstart_training].
 
+To train a model, you will need Azure Blob storage container's shared access signature (SAS) URL. See [here][train-a-model-using-labeled-data] for more detailed descriptions on how to get it.
+Alternatively, a container uri can be used instead if the container is public.
+
```

Comment:
I suggest rebasing on master, I edited this part of the README in a different PR and I'm not sure my edit + your edit make sense now 😆

Reply:
weird - it doesn't show any conflicts even when i rebase it - ill just remove it for now and fix it later